Source/Core/DolphinWX/GameListCtrl.h

```diff
@@ -14,6 +14,9 @@
 
 #include "DolphinWX/ISOFile.h"
 
+// Luminance threshold to determine whether to use black text on light background
+#define LUM_THRESHOLD 186
```

Comment:
Yes, sorry.  I only read that part of your comment after I pushed.  I'm fixing that now... To used to working with plain C.